doc/include/api_stdat_cproc.inc

```diff
@@ -1,6 +1,24 @@
-
+[comment {
+	 Standard argument types for use with
+	 cproc and cclass methods.
+}]
 [list_begin definitions]
 [comment {% % %% %%% %%%%% %%%%%%%% %%%%%%%%%%%%%}]
+
+[def Tcl_Interp*]
+
+[strong Attention]: This is a [strong special] argument type. It can
+[strong only] be used by the [strong first] argument of a function.
+Any other argument using it will cause critcl to throw an error.
+
+[para] When used, the argument will contain a reference to the current
+interpreter the function body may use. Furthermore the argument will
```

Comment:
“interpreter **that** the function body may use”

Reply:
Thanks. Addressed in commit 3e7a30b33a7cae8fc4f1cbd923ec80e4139e2f66